### PR TITLE
refactor: Revise job schemas 

### PR DESCRIPTION
**Pull request by forest1040**
#111 internal ticket

### CODE REVIEW

**Comment by orangekame3** [comment]
@forest1040 
Please make the following corrections

- Resolving Conflict
- Edit PR Title
  - ex) refactor: xxxxxxx 
- Fill in PR Descriptions
- Confirmation (recommended)
- Confirmation of test pass
- Manual confirmation of endpoint response, etc. 

I would appreciate it if you would follow the following instructions for your next commit message.
→[Development Flow \- OQTOPUS](https://oqtopus-cloud.readthedocs.io/latest/developer_guidelines/#conventional-commits)

**Comment by katsujukou** [comment]
…mentation site

# 📃 Ticket

## ✍ Description
Revised the structure of the OpenAPI specification, particularly the definition of the job. This introduces breaking changes, including:

- OAS: The endpoints for controlling quantum jobs have been modified, including:
- Codebase: In accordance with the changes in the OAS, the generated Python data models have also been updated.
- DB: The database schema for the jobs table has been drastically altered.

## 📸 Test Result
Please refer to the results of Github Actions.

## 🔗 Related PRs

File: backend/oas/user/schemas/devices.yaml [hunk 1]
```diff
@@ -50,80 +48,17 @@ devices.DeviceInfo:
       type: string
       example: "State vector-based quantum circuit simulator"
   required: [
-    deviceId, deviceType, status, nPendingTasks, basisGates, supportedInstructions, description
+    deviceId, deviceType, status, nPendingJobs, basisGates, supportedInstructions, description
```
Comment (orangekame3) [inline backend/oas/user/schemas/devices.yaml:51]: should be snake case.

`deviceId, deviceType, status, nPendingJobs, basisGates, supportedInstructions, description`

File: backend/oas/user/schemas/jobs.yaml [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+      "nQubits": 5,
+      "nNodes": 12,
+      "nPerNode": 2,
+      "seedSimulation": 39058567,
+      "simulationOpt": {
+        "optimizationMethod": "light",
+        "optimizationBlockSize": 1,
+        "optimizationSwapLevel": 1
+      }
+    }
```
Comment (orangekame3) [inline backend/oas/user/schemas/jobs.yaml:132]: example also should be snake case.

File: backend/oas/provider/schemas/jobs.yaml [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+    - cancelled
+  example: "submitted"
+
+jobs.JobInfoSampling:
+  type: object
+  description: "The descriptor of sampling jobs"
+  properties:
+    job_type:
+      type: string
+      enum: ["sampling"]
```
Comment (orangekame3) [inline backend/oas/provider/schemas/jobs.yaml:23]: I want the form of enum to be the same as estimation 
- "sampling"

Comment (katsujukou) [reply backend/oas/provider/schemas/jobs.yaml]: I agree and finished to modify. Other enum fields are also modified.

File: backend/oas/user/schemas/devices.yaml (hunk 1 above)
Comment (katsujukou) [reply backend/oas/user/schemas/devices.yaml]: Finished to modify.

**Review by shgokita — APPROVED** [review]
LGTM!

**Review by orangekame3 — APPROVED** [review]
Thanks a lot.
LGTM 🐱

**Pull request merged**